daemon/container_windows.go

```diff
@@ -102,6 +106,26 @@ func populateCommand(c *Container, env []string) error {
 
 	processConfig.Env = env
 
+	var layerFolder string
+	var layerPaths []string
+
+	// The following is specific to the Windows driver. We do this to
+	// enable VFS to continue operating for development purposes.
+	if wd, ok := c.daemon.driver.(*windows.WindowsGraphDriver); ok {
+		var err error
+		var img *graph.Image
+		var ids []string
+
+		if img, err = c.daemon.graph.Get(c.ImageID); err != nil {
+			return fmt.Errorf("Failed to graph.Get on ImageID %s - %s", c.ImageID, err)
+		}
+		if ids, err = c.daemon.graph.ParentLayerIds(img); err != nil {
+			return fmt.Errorf("Failed to get parentlayer ids %s", img.ID)
+		}
+		layerPaths = wd.LayerIdsToPaths(ids)
+		layerFolder = filepath.Join(wd.Info().HomeDir, filepath.Base(c.ID))
```

Comment:
This must be my misunderstanding then.  When I wrote this, I tried to add the struct to the interface, and found that only methods can be added to an interface.  When I tried to access the struct directly without a method, it didn't work, so I was under the impression it was a limitation of Go that required me to have a method to access the struct.  If that's not the case, then it's easy enough to adjust the pattern to whatever allows us to get the information we need and remove the method.

Just for my edification, is there any reason why having a method for accessing the struct is bad?